### PR TITLE
Branch user guide

### PR DESCRIPTION
**Pull request by TohLiYuan**
Fix issue #123

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -80,17 +80,28 @@ Format: `add n/NAME q/QUANTITY u/UNIT`
...
+| PIECE    | pc, pcs, piece, pieces, PIECE, PIECES |
+
 ### Using up ingredients : `use`
 
 Depletes a specified amount of an ingredient from stock.
 
 Format: `use n/NAME [q/QUANTITY] [u/UNIT]`
 
-* If no quantity and unit is provided, the entire stock of the specified ingredient will be depleted.
+* If no quantity or unit is provided, the entire stock of the specified ingredient will be depleted.
```
Comment (prawnzyy) [inline docs/UserGuide.md:104]: I think this part is still and as only when both are missing will it deplete all. Having one missing will lead to invalid command

**Review by prawnzyy — COMMENTED** [review]
LGTM, though for the and or or portion, we probably need to double confirm the functionality

File: docs/UserGuide.md (hunk 1 above)
Comment (TohLiYuan) [reply docs/UserGuide.md:104]: The behaviour is unfortunately inconsistent due to the parsing, as shown in #150 . This was changed if the parsing were to remain the same as the current implementation

Comment (prawnzyy) [reply docs/UserGuide.md:104]: oh ok then lets stick to this.

Comment (prawnzyy) [reply docs/UserGuide.md:104]: will merge

**Pull request merged**